test/remote/gateways/remote_payu_latam_test.rb

```diff
@@ -232,7 +232,7 @@ def test_successful_purchase_no_description
   end
 
   def test_failed_purchase
-    response = @gateway.purchase(@amount, @declined_card, @options)
+    response = @gateway.purchase(@amount, @declined_card)
```

Comment:
I removed `@options` here and below to prevent the `billing_address` field used with `@options` from overwriting the "name" field, which is what triggers a success or failure response.